lintro/ai/json_response.py

```
@@ -70,3 +73,66 @@ def load_json_object(*, content: str) -> dict[str, Any]:
+        content: Raw or fenced JSON model response.
+
+    Returns:
+        Parsed summary payload dictionary.
+
+    Raises:
+        ValueError: When JSON is invalid or not an object.
+    """
+    try:
+        return load_json_object(content=content)
+    except ValueError:
+        raise
```

Ruff (0.15.20)

[warning] Remove exception handler; error is immediately re-raised

(TRY203)